config/locales/candidate_interface/personal_statement.yml

```diff
@@ -4,10 +4,16 @@ en:
       becoming_a_teacher:
         change_action: why do you want to be a teacher?
         label: Why do you want to be a teacher?
+        complete_hint_text_html: |
+          Proofread this section carefully for spelling and grammar. Ask someone you trust for their opinion.
+          <p>Ensure this is all your own work as plagiarism will be penalised.</p>
```

Comment:
Is this an acceptable way to deal with paragraphs?

Reply:
Feels a bit icky.